src/backend/core/models.py

```diff
@@ -452,6 +454,34 @@ def get_abilities(self, user):
         }
 
 
+class Invitation(BaseModel):
+    """User invitation to teams."""
+
+    email = models.EmailField(_("email address"), null=False, blank=False)
+    team = models.ForeignKey(
+        Team,
+        on_delete=models.CASCADE,
+        related_name="invitations",
+    )
+    role = models.CharField(
+        max_length=20, choices=RoleChoices.choices, default=RoleChoices.MEMBER
+    )
+    issuer = models.ForeignKey(
+        User, on_delete=models.SET_NULL, related_name="invitations", null=True
```

Comment:
I set `on_delete` to `SET_NULL` == If a user delete their account, their pending invitations should remain valid. 

Do you agree ?

Reply:
If a user is deleted, I would delete the invitations they made... because it's weird to invite someone to a group you're leaving shortly afterwards?

Reply:
Done !